sdk/identity/identity/src/msal/utils.browser.ts

```diff
@@ -0,0 +1,292 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import * as msalBrowser from "@azure/msal-browser";
```

Comment:
We should have a `utils.common.ts` which has most of the things we need that are not dependent on the native implementation.  See [core-xml](https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/core/core-xml/src) for an example so we can cut down on code duplication